test/e2e/page-objects/pages/test-dapp.ts

```diff
@@ -596,6 +598,13 @@ class TestDapp {
     });
   }
 
+  async assertEip5792SendCallsError(expectedMessage: string) {
```

Comment:
does it make sense to change the name to `check_eip5792SendCallsError` ?

Reply:
sure! I've renamed it to `checkEip5792SendCallsError` just to make it case consistent